pkg/cri/resource-manager/cache/cache.go

```diff
@@ -294,6 +298,20 @@ type Container interface {
 	SetBlockIOClass(string)
 	// GetBlockIOClass returns the BlockIO class for this container.
 	GetBlockIOClass() string
+
+	// SetCRIRequest sets the current pending CRI request of the container.
+	SetCRIRequest(req interface{}) error
+	// GetCRIRequest returns the current pending CRI request of the container.
+	GetCRIRequest() (interface{}, bool)
+	// ClearCRIRequest returns the current pending CRI request of the container.
```

Comment:
Incorrect docstring?

Reply:
Yes it should be `clears and returns the current`. Will fix.